test/jdk/sun/security/ssl/EngineArgs/DebugReportsOneExtraByte.java

```diff
@@ -91,6 +91,7 @@ public class DebugReportsOneExtraByte extends SSLEngineTemplate {
      * Main entry point for this test.
      */
     public static void main(String args[]) throws Exception {
+        SecurityUtils.removeFromDisabledTlsAlgs("TLS_RSA_*");
```

Comment:
You can instead append "TLS_RSA_*" to "TLSv1" on line 105 as this is only needed to be removed when there are command line arguments.

Reply:
Done